.vscode/launch.json

```diff
@@ -9,14 +9,34 @@
       "request": "launch",
       "name": "Run Mocha tests",
       "program": "${workspaceRoot}/packages/build/node_modules/.bin/_mocha",
+      "runtimeArgs": ["-r", "${workspaceRoot}/packages/build/node_modules/source-map-support/register"],
       "cwd": "${workspaceRoot}",
       "args": [
         "--config",
         "${workspaceRoot}/packages/build/config/.mocharc.json",
         "packages/*/dist/__tests__/**/*.js",
         "-t",
         "0"
-      ]
+      ],
+      "windows": {
+        "program": "${workspaceRoot}/packages/build/node_modules/mocha/bin/mocha",
+      }
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Current Test File",
+      "program": "${workspaceRoot}/bin/mocha-current-file",
+      "runtimeArgs": ["-r", "${workspaceRoot}/packages/build/node_modules/source-map-support/register"],
+      "cwd": "${workspaceRoot}",
+      "args": [
+        "--config",
+        "${workspaceRoot}/packages/build/config/.mocharc.json",
+        "-t",
+        "0",
+        "${file}"
+      ],
+      "disableOptimisticBPs": true,
```

Comment:
Can you please add a comment to explain why is this flag needed?

Reply:
Done!